docs/deployment/connect-scm.md

```diff
@@ -0,0 +1,98 @@
+
+The process to connect a source code manager depends on whether your SCM tool is cloud-hosted by the service provider, hosted on-premise, or hosted as a single-tenant by the service provider.
+
+## Connect to cloud-hosted orgs
+
+To connect your SCM:
+
+1. Sign in to Semgrep Cloud Platform.
+1. On the sidebar, click **the organization account** you want to make a connection for.
+1. Click **Settings** > **Source Code Managers**.
+![Source code manager tab](/img/source-code-manager.png#md-width)
+1. For GitHub users, click **Connect to GitHub**. You are taken to the connection page. Enter the following information:
+    1. Click the **Install on** drop-down box and select the account type you are connecting to, either a GitHub **personal account** or **organization account**.
+    1. Under **Organization name**, enter the name of the GitHub account to link to. The name must be an exact match.
+    1. Choose to **<i class="fa-regular fa-circle-dot"></i> Leave PR comments and get Assistant recommendations** or **<i class="fa-regular fa-circle-dot"></i> Leave PR comments**. 
+    1. Optional: Click Review permissions to view the permissions granted to Semgrep.
+    1. Click **Create GitHub App**.
+    1. Click **Install**. 
+1. After a successful link, you are signed out of Semgrep Cloud Platform automatically, as your credentials have changed after linking an organization.
+1. Sign back in to Semgrep Cloud Platform.
+
+:::tip
+- Getting Assistant recommendations grants Semgrep **code access**.
+- **Leave PR comments** refers to Semgrep's capability to post findings to developers in PRs.
+:::
+
+You have successfully connected an org in Semgrep Cloud Platform with an organization in your source code management tool.
+
+## Connect to on-premise GitHub or GitLab orgs
+
+### Applicable plans
+
+| GitHub | GitLab |
+| ------ | ------ |
+| GitHub Enterprise Server       | Any GitLab Self Managed plan        |
+
+### Table of required scopes for PATs
+
+Semgrep Cloud Platform requires personal access tokens (PATs) with assigned scopes. These scopes grant necessary permissions to the PAT and vary depending on the user's plan.
+
+| GitHub Enterprise Server          | GitLab Self-Managed        |
+|:---------------------------|:---------------------------|
+| <ul><li>`public_repo`</li> <li>`repo:status`</li> <li>`user:email`</li> <li>`write:discussion`</li></ul> | `api` |
+
+### Make the connection
+
+Integrate Semgrep into these custom source code management (SCM) tools by following the steps below:
+
+1. Sign in to [Semgrep Cloud Platform](https://semgrep.dev/login).
+2. Click **Settings** > **Source Code Managers**.
+![Source code manager tab](/img/source-code-manager.png#md-width)
+3. Select your source code manager.
+4. For **GitHub Enterprise Server**, follow these steps:
```

Comment:
Do we want to consider detailing steps for the custom GHES app instead of using the old way via PAT?

Reply:
Chris's team owns this (F&S) - the "new way" is in beta for now. I'll resolve this and return to this section once the new way is GA.